Sources/XKit/DeveloperServices/OpenAPI/ASCKey.swift

```diff
@@ -16,7 +16,10 @@ public struct ASCKey: Sendable {
 actor ASCJWTGenerator {
     // the duration for which we generate JWTs.
     // ASC allows a maximum of 20 minutes.
-    private static let ttl: TimeInterval = 60 * 20
+    //
+    // Apple sometimes rounds up, causing the expiry to go over 20 seconds.
+    // This leads to an invalid token.
+    private static let ttl: TimeInterval = 60 * 19
```

Comment:
Let's remove this from the current PR before landing it, would rather split it out as discussed

Reply:
Alright. I have created a new pull request: https://github.com/xtool-org/xtool/pull/129